packages/app/src/cli/models/extensions/ui-specifications/tax_calculation.ts

```diff
@@ -17,6 +17,7 @@ const spec = createUIExtensionSpecification({
       production_api_base_url: config.productionApiBaseUrl,
       benchmark_api_base_url: config.benchmarkApiBaseUrl,
       calculate_taxes_api_endpoint: config.calculateTaxesApiEndpoint,
+      metafields: config.metafields
```

Comment:
I believe you also need to add this in line 8 above since it is part of the schema.